### PR TITLE
[FIX] web_editor: fix layout issue

### PR DESCRIPTION
**Pull request by dep-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web_editor/static/src/xml/editor.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
     <!-- Media Dialog (allows to choose an img/pictogram/video) -->
     <div t-name="web_editor.dialog.media">
         <ul class="nav nav-tabs" role="tablist">
-            <li class="nav-item"><a role="tab" href="#editor-media-image" aria-controls="editor-media-image" data-toggle="tab" t-attf-class="nav-link#{widget.noImages ? ' d-none' : ''}">Image</a></li>
+            <li class="nav-item"><a role="tab" href="#editor-media-image" aria-controls="editor-media-image" data-toggle="tab" t-attf-class="nav-link active#{widget.noImages ? ' d-none' : ''}">Image</a></li>
```
Comment (qsm-odoo) [inline addons/web_editor/static/src/xml/editor.xml]: Supposed to be automatically active thanks to https://github.com/odoo/odoo/blob/master/addons/web_editor/static/src/js/widgets/widgets.js#L1157 -> the fix would be to check why it is not the case

File: addons/web_editor/static/src/xml/editor.xml [hunk 2]
```diff
@@ -84,8 +84,7 @@
             <input type="hidden" name="filters" t-att-value="widget.firstFilters.join('_')"/>
             <input t-if="widget.options.res_id" type="hidden" name="res_id" t-att-value="widget.options.res_id"/>
             <input t-if="widget.options.res_model" type="hidden" name="res_model" t-att-value="widget.options.res_model"/>
-            <div class="card">
-                <div class="card-body">
+            <div class="card card-body">
```
Comment (qsm-odoo) [inline addons/web_editor/static/src/xml/editor.xml]: No :wink:

File: addons/web_editor/static/src/xml/editor.xml (hunk 1 above)
Comment (dep-odoo) [reply addons/web_editor/static/src/xml/editor.xml]: @qsm-odoo 
there is only one case when we click in image button in the top panel ( https://drive.google.com/file/d/1PZEtCLqVTQkVcJihHxD4GaTF2SIOjYfH/view?usp=drivesdk ) then image tab-panel is not active.

**Comment by qsm-odoo** [comment]
Merged https://github.com/odoo/odoo/commit/08117383d06d5fc3195eab47de6462fc4863c747

**Pull request closed without merging**